src/GeneralStateTestsFiller/stExample/03_tloadAfterStoreIs0Filler.yml

```diff
@@ -0,0 +1,63 @@
+# 03
+# Loading any other slot after storing to a slot returns 0: TSTORE(x, y), TLOAD(x + n) where n > 0 returns 0
```

Comment:
hm. from math point of view you have to loop from 1 to 2**256 to prove that. just noting. I dont think it is reasonable.